DigitalOffice/Tests/UserServiceTests/MappersTests/UserCreateRequestToDbUserMapperTests.cs

```diff
@@ -6,7 +6,7 @@
 using LT.DigitalOffice.UserService.Mappers;
 using LT.DigitalOffice.UserService.RestRequests;
 
-namespace LT.DigitalOffice.UserServiceUnitTests.Mappers
+namespace LT.DigitalOffice.UserServiceUnitTests.MappersTests
```

Comment:
It is necessary to change the name from MappersTests to Mappers.

Reply:
Fixed